src/server/raft_client.rs

```diff
@@ -52,8 +50,7 @@ impl Conn {
 
         let cb = ChannelBuilder::new(env)
             .stream_initial_window_size(cfg.grpc_stream_initial_window_size.0 as i32)
-            .max_receive_message_len(MAX_GRPC_RECV_MSG_LEN)
```

Comment:
Does this mean the max receive len is not limited or gRPC has it's default value?